src/platform/silabs/wifi/wf200/WifiInterfaceImpl.cpp

```diff
@@ -30,6 +30,7 @@
 #include <lib/support/CHIPMemString.h>
 #include <lib/support/CodeUtils.h>
 #include <lib/support/logging/CHIPLogging.h>
+#include <platform/silabs/wifi/WifiInterface.h>
```

Comment:
This inlcude should not have any impact since it is available through the WifiInterfaceImpl header.